packages/firebase-frameworks/package.json

```
@@ -53,7 +53,7 @@
     "tslib": "^2.3.1"
   },
   "peerDependencies": {
-    "firebase": "^9.9.0 || ^10.0.0",
+    "firebase": "^9.9.0 || ^11.0.0",
```

we still need to support v10
```suggestion
    "firebase": "^9.9.0 || ^10.0.0 || ^11.0.0",
```